### PR TITLE
fix(FLUXDOC.md): add path to PackageDoc spec

### PR DESCRIPTION
**Pull request by BarWeiner1**
### Done checklist
- [ ] docs/SPEC.md updated
- [ ] Test cases written

### CODE REVIEW

File: libflux/flux/FLUXDOC.md [hunk 1]
```diff
@@ -354,6 +354,7 @@ enum Doc {
 }
 
 struct PackageDoc {
+       path: String,
 	name: String,
```
Comment (scbrickley) [inline libflux/flux/FLUXDOC.md:358]: Can we fix the formatting here?

File: stdlib/contrib/sranka/webexteams/webexteams.flux [hunk 2]
```diff
@@ -35,10 +37,7 @@ message = (
 // `token` - string - [Webex API access token](https://developer.webex.com/docs/api/getting-started).
 // The returned factory function accepts a `mapFn` parameter.
 // The `mapFn` must return an object with `roomId`, `text` and `markdown` properties  as defined in the `message` function arguments.
-endpoint = (
-        url = "https://webexapis.com",
-        token,
-) => (mapFn) => (tables=<-) => tables
+endpoint = (url="https://webexapis.com", token) => (mapFn) => (tables=<-) => tables
```
Comment (scbrickley) [inline stdlib/contrib/sranka/webexteams/webexteams.flux:40]: Were the changes in this file just formatter changes? They don't seem relevant to the rest of the PR.

File: libflux/flux/FLUXDOC.md (hunk 1 above)
Comment (scbrickley) [reply libflux/flux/FLUXDOC.md:358]: Also, how are we populating the path when parsing? Is this field just going unused for now?

**Review by scbrickley — CHANGES_REQUESTED** [review]

File: stdlib/contrib/sranka/webexteams/webexteams.flux (hunk 2 above)
Comment (BarWeiner1) [reply stdlib/contrib/sranka/webexteams/webexteams.flux:40]: Yes this was formatting

File: libflux/flux/FLUXDOC.md (hunk 1 above)
Comment (scbrickley) [reply libflux/flux/FLUXDOC.md:358]: Just realized this is just the example code in the spec. I would prefer if changes to the spec were accompanied by the appropriate changes to the code. Can we include that as part of this PR?

Comment (BarWeiner1) [reply libflux/flux/FLUXDOC.md:358]: What exactly is wrong with the formatting? Check out the most recent parser changes, but basically we are using the PolyTypeMap to pass in the relative path when populating.

Comment (scbrickley) [reply libflux/flux/FLUXDOC.md:358]: Just the indentation. The fields in a struct should all be aligned.

Did we already do the work to include the path in `PackageDoc`? If so, then disregard my last comment.

Comment (BarWeiner1) [reply libflux/flux/FLUXDOC.md]: Does this look better, it is rendering very weird on my device

**Review by scbrickley — APPROVED** [review]
